src/python/pywraps2_test.py

```diff
@@ -951,5 +951,185 @@ def testS2CellIdFromDebugString(self):
     self.assertTrue(cell.is_valid())
     self.assertEqual("5/31200", cell.ToString())
 
+class ChordAngleTest(unittest.TestCase):
```

Comment:
Having test classes is consistent with the rest of the file, but I like it. This would normally be called `S2ChordAngleTest`

Reply:
This is normal in Python unittest code, in my experience at least.

I chose the names following the example of `RegionTermIndexerTest`, but I'm happy to change them.